src/jit/ssabuilder.cpp

```diff
@@ -232,17 +151,85 @@ SsaBuilder::SsaBuilder(Compiler* pCompiler, IAllocator* pIAllocator)
  */
 int SsaBuilder::TopologicalSort(BasicBlock** postOrder, int count)
```

Comment:
It would be nice to update the function header according to the coding conventions.